scapy/arch/windows/__init__.py

```diff
@@ -905,7 +905,7 @@ def _get_valid_guid():
     if scapy.consts.LOOPBACK_INTERFACE:
         return scapy.consts.LOOPBACK_INTERFACE.guid
     else:
-        for i in IFACES:
+        for i in six.itervalues(IFACES):
```

Comment:
How does it come that you can replace an iteration over `IFACES` with an iteration over `six.itervalues(IFACES)`? Shouldn't it be `six.iterkeys(IFACES)`?

Reply:
No, the keys of IFACES are the GUIDs of the NetworkInterface. On Python 2, the loop would iterate over the values, on python 3, it's over the keys.